Assets/MixedRealityToolkit.SDK/Features/Utilities/Solvers/SolverHandler.cs

```diff
@@ -134,17 +176,62 @@ public bool UpdateSolvers
         /// </summary>
         public float DeltaTime { get; set; }
 
-        private bool RequiresOffset => AdditionalOffset.sqrMagnitude != 0 || AdditionalRotation.sqrMagnitude != 0;
+        /// <summary>
+        /// The target transform that the solvers will act upon.
+        /// </summary>
+        public Transform TransformTarget
+        {
+            get
+            {
+                if (IsInvalidTracking())
+                {
+                    RefreshTrackedObject();
+                }
 
-        protected readonly List<Solver> solvers = new List<Solver>();
+                return trackingTarget?.transform;
+            }
+        }
 
-        private float lastUpdateTime;
 
-        private GameObject transformWithOffset;
+        // Stores currently attached hand if valid (only possible values Left, Right, or None)
+        protected Handedness currentTrackedHandedness = Handedness.None;
+
+        /// <summary>
+        /// Currently tracked hand or motion controller if applicable
```

Comment:
>Currently tracked hand or motion controller if applicable [](start = 12, length = 57)

Nit: Add a remarks section which contains the comments from above:

These are super valuable to have in autocomplete

// Stores currently attached hand if valid (only possible values Left, Right, or None)